packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -1045,6 +1045,18 @@ class ScaffoldState extends State<Scaffold> with TickerProviderStateMixin {
   bool _drawerOpened = false;
   bool _endDrawerOpened = false;
 
+  /// Whether the [Scaffold.drawer] is opened.
+  ///
+  /// See also:
+  ///   * https://docs.flutter.io/flutter/material/ScaffoldState/openDrawer.html
```

Comment:
This line should probably be:
`* [ScaffoldState.openDrawer], explanation why I should look there`